src/content-tags-drawer/ContentTagsCollapsible.test.jsx

```diff
@@ -18,8 +19,12 @@ jest.mock('./data/apiHooks', () => ({
     mutate: jest.fn(),
   })),
   useTaxonomyTagsData: jest.fn(() => ({
-    isSuccess: false,
-    data: {},
+    hasMorePages: false,
+    tagPages: [{
+      isLoading: true,
+      isError: false,
+      data: [],
+    }],
```

Comment:
Personally, I find it easier to mock the REST API, e.g. `axiosMock.onGet(rootTagsListUrl).reply(200, mockTagsResponse);` as you can see in `TagListTable.test.jsx`. Don't change what you have now, but if you have trouble with these hook mocks in the future, I'd definitely recommend going that way instead (mock the REST API endpoints, not the hooks). The other advantage that has is that it tests the functionality of your hooks instead of mocking them.

Reply:
Noted, I actually use this approach in the `api.test.js`. It would definitely be helpful here as well.